readyapi4j-core/src/main/java/com/smartbear/readyapi4j/teststeps/mockresponse/SoapMockResponseTestStepBuilder.java

```diff
@@ -1,47 +1,47 @@
 package com.smartbear.readyapi4j.teststeps.mockresponse;
 
-import com.smartbear.readyapi.client.model.WsdlMockResponseTestStep;
+import com.smartbear.readyapi.client.model.SOAPMockResponseTestStep;
 import com.smartbear.readyapi4j.teststeps.TestStepBuilder;
 import com.smartbear.readyapi4j.teststeps.TestStepTypes;
 
 import java.net.URL;
 
-public class WsdlMockResponseTestStepBuilder implements TestStepBuilder<WsdlMockResponseTestStep> {
-    private final WsdlMockResponseTestStep mockResponseTestStep = new WsdlMockResponseTestStep();
+public class SoapMockResponseTestStepBuilder implements TestStepBuilder<SOAPMockResponseTestStep> {
+    private final SOAPMockResponseTestStep mockResponseTestStep = new SOAPMockResponseTestStep();
```

Comment:
Well, in recipes we have "type: SOAPMockResponse" (which I think makes sense) and this is auto generated code, hence it is all caps here, but otherwise we have Soap in all other manually created java classes.